src/web/app/components/question-types/question-edit-answer-form/num-scale-question-edit-answer-form.component.html

```diff
@@ -1,9 +1,13 @@
 <div class="row">
   <div class="col-5">
-    <input type="number" class="form-control" min="{{ questionDetails.minScale }}" max="{{ questionDetails.maxScale }}" step="{{ questionDetails.step }}"
+    <input type="number" class="form-control" min="{{ questionDetails.minScale }}" max="{{ questionDetails.maxScale }}" step="{{ questionDetails.step }}" #scale
            [ngModel]="responseDetails.answer == NUMERICAL_SCALE_ANSWER_NOT_SUBMITTED ? '' : responseDetails.answer" (ngModelChange)="triggerResponseDetailsChange('answer', $event)" [disabled]="isDisabled">
   </div>
   <div>
     Possible Acceptable Values are: {{ possibleValues }}
   </div>
+  <div>
+    <p *ngIf="!isValidPossibleValue(scale.value) && scale.value != '0'" style="margin-left: 1em;" class="text-danger">The scale entered is not a possible value.</p>
```

Comment:
You should use `responseDetails` (MODEL, Angular way) rather than reference HTML element directly (JQuery-like way).

Reply:
I see. I have updated it.